docs/install.md

```diff
@@ -22,6 +22,16 @@ To be able to clone the repo and run the application you need:
 
 ## Installing and Running
 
+### Ubuntu and [snap supported distros](https://snapcraft.io/docs/core/install) 
```

Comment:
Might it be better to have this set of instructions follow the `From Source` instructions.
I would imagine the existing instructions would serve more folks and as such should take some precedence in the documents hierarchy.

Any thoughts? cc: @johngodley @astralbodies?